challenges/objects-arrays.js

```diff
@@ -1,17 +1,38 @@
 // ==== Objects ====
 
-/* 
-  Given the following information about dinosaurs, create 3 objects: 
-  Use this pattern to create your objects: 
+/*
+  Given the following information about dinosaurs, create 3 objects:
+  Use this pattern to create your objects:
   object name, diet, weight, length, period
 */
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
-
+const tyrannosaurus = {
+  name: "T-Rex",
+  diet: 'carnivorous',
+  weight: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceious',
+  roar: function() {
```

Comment:
Declare roar outside of the tyrannosaurus object declaration.

Reply:
Check into this article on method definitions to make your method syntax shorter: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions

:)